test/e2e/specs/specs-wpcom/wp-widgets__spec.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @group gutenberg
+ */
+
+import {
+	BrowserHelper,
+	DataHelper,
+	SidebarComponent,
+	setupHooks,
+	TestAccount,
+	BlockWidgetEditorComponent,
+	skipDescribeIf,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+const accountName = BrowserHelper.targetGutenbergEdge()
+	? 'gutenbergSimpleSiteEdgeUser'
+	: 'gutenbergSimpleSiteUser';
```

Comment:
If my memory doesn't fail, we recently changed those users due to problems related to the nux dialogs, no?

Reply:
You are correct, for Calypso tests I switched it over to `simpleSitePersonalPlanUser`. 

Sadly it turns out the nux dialog is an ongoing issue with basically all test accounts that touch the editor in some form or another, so it wasn't the solution I had hoped it would be. 

I'll keep the user as is for now, as the test appears to pass as expected. At _some point_ I plan to do a secrets file clean sweep.